### PR TITLE
Add Mazda breaking changes to 2022.4 release notes

### PR DESCRIPTION
**Pull request by bdr99**
## Proposed change

It looks like the breaking changes to the Mazda integration from https://github.com/home-assistant/core/pull/67597 and https://github.com/home-assistant/core/pull/68025 were missed in the 2022.4 release notes. This PR adds them so that users will be informed of the deprecated services.

Hope it's OK to open a PR targeted at the `rc` branch. I waited a few days to see if this would be fixed, but now we are 1 day from release and it doesn't look like it will be fixed.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by frenck** [comment]
Neither of the PRs are marked as breaking change, hence they didn't end up in the breaking changes: 

CleanShot 2022-04-05 at 23 43 13@2x

**Comment by bdr99** [comment]
@frenck Apologies, I must have forgotten to check that box when creating the PRs. Is it too late to still add this to the release notes?

**Comment by frenck** [comment]
No, its never too late 😄 

I've tagged  the PR for the milestone,  will be take care of.

**Comment by bdr99** [comment]
Great! Thanks 😁

File: source/_posts/2022-04-06-release-20224.markdown [hunk 1]
```diff
@@ -1023,6 +1023,29 @@ removed from your YAML configuration files.
 
 {% enddetails %}
 
+{% details "Mazda" %}
+
+The Mazda services `start_engine`, `stop_engine`, `turn_on_hazard_lights`,
+and turn_off_hazard_lights are deprecated and have been replaced by button
```
Comment (frenck) [inline source/_posts/2022-04-06-release-20224.markdown]:
```suggestion
and `turn_off_hazard_lights` are deprecated and have been replaced by button
```

**Review by frenck — APPROVED** [review]
Thanks, @bdr99 👍

**Pull request merged**